### PR TITLE
上传英文版快速推理使用指引

### PR DESCRIPTION
**Pull request by wsl-iei**
上传英文版本的推理使用指导

### CODE REVIEW

**Comment by Shawn-IEITSystems** [comment]
@Huifu1018 please review

File: docs/quickstart_inference.md [hunk 1]
```diff
@@ -1,52 +1,63 @@
-# 快速推理指引
+# Quick Start Inference
```
Comment (Huifu1018) [inline docs/quickstart_inference.md:1]: Quick Start: Inference

File: docs/quickstart_inference.md [hunk 2]
```diff
@@ -1,52 +1,63 @@
...
 
-本脚本介绍了102B模型和51B模型快速使用指引，主要包括ckpt转换以及推理服务使用
+This script describes the inference guide of the 102B model and the 51B model, including the ckpt conversion and the use of the inference service.
```
Comment (Huifu1018) [inline docs/quickstart_inference.md:3]: This script provides a quick guide on using the 102B model and the 51B model, including instructions for checkpoint (ckpt) conversion and utilizing inference services.

File: docs/quickstart_inference.md [hunk 3]
```diff
@@ -1,52 +1,63 @@
...
 
-## 102B模型：
+## 102B：
```
Comment (Huifu1018) [inline docs/quickstart_inference.md:5]: Yuan 2.0-102B

Comment (wsl-iei) [reply docs/quickstart_inference.md]: 已经修改，麻烦review一下吧

**Comment by Shawn-IEITSystems** [comment]
LGTM.

**Pull request merged**